django-cloudlaunch/cloudlaunch/backend_plugins/cloudman2/cloudman2_app.py

```diff
@@ -391,26 +401,44 @@ def _gen_cloud_conf(self, provider_id, cloud_config):
             conf_template = GCP_CLOUD_CONF
             values = {}
         elif provider_id == "openstack":
-            # http://henriquetruta.github.io/openstack-cloud-provider/
-            conf_template = OPENSTACK_CLOUD_CONF
-            os_ignore_az = self._os_ignore_az(
-                zone.get('zone_id'),
-                zone.get('region', {}).get('cloudbridge_settings'))
-            if creds.get('os_user_domain_id'):
-                domain_entry = f"domain-id={creds.get('os_user_domain_id')}"
+            os_user = creds.get('os_username')
```

Comment:
Will `os_username` and `os_password` still work on Jetstream2?  In my `openrc` file they have been replaced with
`OS_APPLICATION_CREDENTIAL_ID` and `OS_APPLICATION_CREDENTIAL_SECRET`

Reply:
They will not on JS2, the point of this PR is to add support for application credentials. However, given that this is openstack in general, not just for JS2, it's good to leave both options, so other places, eg: Nectar, would still work with user/pass. A few lines below, there is a check for whether both username and password are passed, and otherwise it falls back on application credentials.

Reply:
NeCTAR also supports app creds. What if we drop the legacy creds in cloudlaunch, and just stick to app creds only? The UI would certainly become simpler. We can leave things as is in cloudbridge, since we don't want an interface change, but for cloudlaunch, we may as well? Would need to test complete flow through cloudman including adding nodes though. Something we may need to do regardless?

Reply:
Ok I don't mind sticking to only app creds then